### PR TITLE
Feedback promo

### PR DESCRIPTION
**Pull request by vauxhall**
*Thank you for submitting a pull request, your contributions are greatly appreciated!*

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Please make sure you've run the test scheme and all the tests pass. If your changes affect existing tests please make sure to update the tests as well.

### CODE REVIEW

File: Client/Frontend/Home/FirefoxHomeViewController.swift [hunk 1]
```diff
@@ -173,26 +180,47 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
         self.collectionView?.register(ASHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: "Header")
         self.collectionView?.register(ASFooterView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionFooter, withReuseIdentifier: "Footer")
         collectionView?.keyboardDismissMode = .onDrag
-
-        if #available(iOS 14.0, *), !UserDefaults.standard.bool(forKey: "DidDismissDefaultBrowserCard") {
-            self.view.addSubview(defaultBrowserCard)
-            defaultBrowserCard.snp.makeConstraints { make in
+        
+        if true || User.shared.showsFeedbackPromo {
```
Comment (vauxhall) [inline Client/Frontend/Home/FirefoxHomeViewController.swift:184]: Temporally forcing the promo to be visible.

File: Client.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -6050,10 +6052,11 @@
 		F84B22211A09122500AAB793 /* Home */ = {
 			isa = PBXGroup;
 			children = (
-				74821F8D1DAD8F1300EEEA72 /* FirefoxHomeHighlightCell.swift */,
-				3BB50E1F1D627539004B33DF /* FirefoxHomeViewController.swift */,
 				2386E4E524F8358E0072EF17 /* DefaultBrowserCard.swift */,
+				CE4C3DC325ED41E90049E9BF /* FeedbackCard.swift */,
```
Comment (ecotopian) [inline Client.xcodeproj/project.pbxproj:6056]: It's a new file, isn't it? I'd say let's put into our Ecosia folder to have less potential conflicts in future merges of the project file and for house-keeping, don't you think?

File: Client/Frontend/Home/FeedbackCard.swift [hunk 3]
```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// TODO: Remove before going to production
```
Comment (ecotopian) [inline Client/Frontend/Home/FeedbackCard.swift:5]: Let's hope we remember this 🤞

File: Client/Frontend/Home/FeedbackCard.swift [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+import Core
+
+final class FeedbackCard: UIView {
+    public var dismissClosure: (() -> Void)?
+    lazy var title: UILabel = {
+        let title = UILabel()
+        title.text = .localized(.helpUsImprove)
+        title.numberOfLines = 0
+        title.lineBreakMode = .byWordWrapping
+        title.font = UIFont.systemFont(ofSize: 20, weight: .bold)
```
Comment (ecotopian) [inline Client/Frontend/Home/FeedbackCard.swift]: Usually I'd say let's stick to the way FF does dynamic fonts, but as it's only for a short living feature it's ok to leave it :)

Comment (vauxhall) [reply Client/Frontend/Home/FeedbackCard.swift]: I didn't change fonts here, I copy/pasted what it is being used on their own promo.

File: Client/Frontend/Settings/AppSettingsOptions.swift [hunk 5]
```diff
@@ -607,6 +607,18 @@ class ToggleNewTabToolbarButton: HiddenSetting {
     }
 }
 
+final class PushBackInstallation: HiddenSetting {
```
Comment (ecotopian) [inline Client/Frontend/Settings/AppSettingsOptions.swift:610]: Cool 😎

File: Client/Frontend/Settings/AppSettingsTableViewController.swift [hunk 6]
```diff
@@ -140,21 +140,8 @@ class AppSettingsTableViewController: SettingsTableViewController {
             SettingSection(title: NSAttributedString(string: NSLocalizedString("About", comment: "About settings section title")), children: [
                 VersionSetting(settings: self),
                 LicenseAndAcknowledgementsSetting(),
-                // Ecosia: YourRightsSetting(),
```
Comment (ecotopian) [inline Client/Frontend/Settings/AppSettingsTableViewController.swift:143]: what about a block comment here?

**Review by ecotopian — APPROVED** [review]
promoted! 🥇

**Pull request merged**